composer.json

```diff
@@ -21,6 +21,7 @@
     "require": {
         "php": "^5.6 || ^7.1",
         "ezsystems/ezpublish-kernel": "^6.13@dev || ^7.5@dev",
+        "ezsystems/ezplatform-http-cache": "~0.9.0",
```

Comment:
Please use `>=0.9, <1.0` as a constraint so we can support future versions of HTTP Cache lib too.